contracts/interfaces/IRelayVerifier.sol

```diff
@@ -20,4 +20,6 @@ interface IRelayVerifier {
         EnvelopingTypes.RelayRequest calldata relayRequest,
         bytes calldata signature
     ) external returns (bytes memory context);
+
+    event AllowedToken(address indexed tokenAddress);
```

Comment:
There is no need to add events to determine which tokens are accepted, the relay server can query the state of the contract directly and that is an off-chain call without any cost. This scales better than the use of events. For simplicity, the verifiers can have a list of accepted tokens (additionally to the current map) so parsing the state is easier.